airflow/api_fastapi/execution_api/schemas.py

```diff
@@ -119,3 +119,30 @@ class TIHeartbeatInfo(BaseModel):
 
     hostname: str
     pid: int
+
+
+class ConnectionResponse(BaseModel):
+    """Connection serializer for responses."""
+
+    conn_id: str
+    conn_type: str
+    description: str | None
+    host: str | None
+    schema_: str | None = Field(alias="schema")
```

Comment:
Can we fix this now and make this be called path?

Reply:
https://github.com/orgs/apache/projects/405/views/1?pane=issue&itemId=86397744